conmon/conmon.c

```diff
@@ -399,6 +400,9 @@ static int write_k8s_log(int fd, stdpipe_t pipe, const char *buf, ssize_t buflen
 		if ((opt_log_size_max > 0) && (bytes_written + bytes_to_be_written) > opt_log_size_max) {
 			bytes_written = 0;
 
+			if (writev_buffer_flush(fd, &bufv) < 0) {
+				nwarn("failed to flush buffer to log");
```

Comment:
should we `goto next;` here as well? or exit altogether if this fails? if this fails, we don't want to keep logging and have a corrupted log file right? @mrunalp wdyt?

Reply:
the safest thing to do probably is to discard any data in the buffer when the flush fails, as anyway the file is going to be deleted soon afterwards.

Reply:
I've added a patch instead of fixing the commit as for the backport I already referenced the checksum of the two first commits.  Also, if we decide to backport the new patch as well, it will be easier to cherry-pick it

Reply:
Sounds good. Thanks!